### DISCUSSION TITLE
create index failed, collection is loaded, please release it first

### DISCUSSION THREAD

**Discussion by zhenzi0322** [milvus-io/milvus#21306] [Q&A and General discussion]
我在使用milvus2.2.0版本进行导入图片到集合中出现如下问题：
`create index failed, collection is loaded, please release it first`。

示例：
```python
...
milvus.insert_vectors_ids(collection_name=table_name, vectors=vectors, ids=ids)
milvus.create_index(table_name)
...
```

**Comment by yhmo** [comment]
从2.2版本开始，要求不能在已load的状态下改变索引，你必须：
建索引 ---> load() ---> release() ----> 改建另一种索引 ---> load()
